lib/utils.ts

```diff
@@ -93,3 +94,11 @@ export const navLinks = [
 export const formatDate = (date: Date) => {
   return format(date, "MMM dd, yyyy");
 };
+
+export function contributorRankClasses(rank: number | null): string {
+  return clsx(
+    rank != null && rank > 0 && rank <= 3
+      ? `${["text-yellow-600", "text-stone-600", "text-amber-700"][rank - 1]} animate-circular-shadow`
+      : "text-purple-600",
+  );
+}
```

Comment:
This code code be made more readable.

```suggestion
export function contributorRankClasses(rank: number | null): string {
  if (!rank || rank > 3) {
    return "text-purple-500";
  }
  
  const rankColor = ["text-yellow-600", "text-stone-600", "text-amber-700"][rank - 1];
  return `${rankColor} animate-circular-shadow`;
}
```

Reply:
Yup, It's looks more readable and easy to understand, Thanks for suggestion

Reply:
Will merge once this is pushed

Reply:
Finally merge it after a week 🫡